CHANGES.md

```diff
@@ -62,6 +62,8 @@ To be released.
     throws `SwarmException`.  [[#694]]
  -  `Swarm<T>` became to print less logs on debug level during sending states.
     [[#706]]
+ -  Increased `Swarm<T>`'s network timeout value, in order to stabilize
```

Comment:
It would be better if we write its previous value (3s) and the current value (30s) as well.

```suggestion
 -  Increased `Swarm<T>`'s network timeout value, in order to be stable
```